schemas/create-status.yml

```diff
@@ -0,0 +1,22 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "Create status"
+description: |
+  Create a commit status on GitHub
+type:         object
+properties:
+  state:
+    type: string
+    enum: ['pending', 'success', 'error', 'failure']
+    description: github status associated with the build.
+  target_url:
+    type: string
+    description: link to put as github status.
+  description:
+    type: string
+    description: title displayed on the github status.
+  context:
+    type: string
+    description: string label to differenciate github statuses.
+additionalProperties: false
+required:
+  - state
```

Comment:
Yes, state is the only mandatory attribute.
I'm defaulting context to "default" to match their API behavior.